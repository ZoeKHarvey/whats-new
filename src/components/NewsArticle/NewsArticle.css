div img {
  width: 100%;
}

/* .news { 
  display: inline-flex;
  flex-direction: column;
  background-color: white;
  width: 40%;
  margin-right: 5%;
  margin-bottom: 5%;
} */

.news {
  background-color: white;
  margin-right: 5%;
  margin-bottom: 5%;
}

  

.article__img {
  /* width: 100%; */
  height: 37vh;
  overflow: hidden;

}

h3 {
  font-size: 2em;
  margin-top: 5%;
}

p {
  font-size: 1em;
  margin: 3%;
}

footer {
  height: 7vh;
  text-align: initial;
  padding-left: 16%;
  background-color: #B56AF0;
  color: white;
  display: flex;
  justify-content: space-between
}

footer:hover {
  font-size: 1em;
  font-weight: bold;
  cursor: pointer;
}

footer p {
  font-size:21px;
}

.img__arrow {
  width: 11%;
  margin-right: 18%
}



