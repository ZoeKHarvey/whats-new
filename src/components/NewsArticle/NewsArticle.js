import React from 'react';
import './NewsArticle.css';

const NewsArticle = () => {
  return (
    <div className="news">
      <h3>News!</h3>
    </div>
  )
}

export default NewsArticle;