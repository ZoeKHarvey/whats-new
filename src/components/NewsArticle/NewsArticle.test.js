import React from 'react';
import { shallow } from 'enzyme';
import NewsArticle from './NewsArticle';

describe('NewsArticle', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(
      <NewsArticle
        key={1}
        id={1}
        headline="This headline"
        image="This Image"
        description="This description"
        url="https://www.thetests.com"
      />
    )
    expect(wrapper).toMatchSnapshot();
  })
})