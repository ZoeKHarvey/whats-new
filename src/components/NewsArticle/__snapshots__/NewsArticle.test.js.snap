// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should match the snapshot with all data passed in correctly 1`] = `
<article
  className="news"
>
  <div
    className="article__img"
  >
    <img
      src="This Image"
    />
  </div>
  <h3>
    This headline
  </h3>
  <p
    className="description__p"
  >
    This description
  </p>
  <a
    className="footer__p"
    href="https://www.thetests.com"
  >
    Link To Article&gt;
    <footer>
      <p
        className="footer__p"
      >
        Link To Article
      </p>
      <img
        className="img__arrow"
        src="right-arrow.svg"
      />
    </footer>
  </a>
</article>
`;
